viz/metrics-api/stat_summary.go

```diff
@@ -897,10 +915,80 @@ func (s *grpcServer) getPolicyMetrics(ctx context.Context, req *pb.StatSummaryRe
 		return nil, nil, err
 	}
 
-	basicStats, tcpStats := processPrometheusMetrics(req, results, groupBy)
+	basicStats, tcpStats := processPolicyPrometheusMetrics(results, groupBy)
 	return basicStats, tcpStats, nil
 }
 
+func processPolicyPrometheusMetrics(results []promResult, groupBy model.LabelNames) (map[sazKey]*pb.BasicStats, map[sazKey]*pb.TcpStats) {
+	basicStats := make(map[sazKey]*pb.BasicStats)
```

Comment:
Tried not to repeat this but using a different key meant that we had to do this and different key had to be used not only to differentiate `Server` and `ServerAuthorization` resources but also to have a different `metricToKey` func which was important to get the queries and results right.

I'm sure this can be simplified but it could be a significant change (which I think we should definitely take up after this)